src/utils/notifications.ts

```diff
@@ -0,0 +1,376 @@
+import { Flat } from 'lodash'
+import { EntityManager } from 'typeorm'
+import {
+  Account,
+  EventData,
+  NextEntityId,
+  NotificationType,
+  OffChainNotification,
+  OffChainNotificationData,
+} from '../model'
+import { ConfigVariable, config } from './config'
+import { MailNotifier } from './mail'
+
+export type NotificationPreferences = {
+  commentCreatedInAppNotificationEnabled: boolean
```

Comment:
codestyle thing: usually when you'll have such large lists, its better to sort the keys in the alphabet order. There's even a prettier\eslint rule of it somewhere, but I think it doesn't worth to include it, lets just sort this one for better readability